### DISCUSSION TITLE
not able to use local postgres database in prisma 2

### DISCUSSION THREAD

**Discussion by rajneesh3923** [prisma/prisma#2394] [Q&A] [answered]
I am trying to use my local postgres database in prisma 2 but it is throwing some error when is try to send the query. Below is the error I get in console

error: Error validating source `db`: The URL for datasource `db` must start with the protocol `postgresql://`.
  -->  schema.prisma:3
   |
 2 |   provider = "postgresql"
 3 |   url      = env("DATABASE_URL")
   |

Validation Error Count: 1
    at PrismaClientFetcher.request (D:\GraphQL\graphql-basics\node_modules\@prisma\client\src\runtime\getPrismaClient.ts:652:34)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

**Comment by ryands17** [comment, marked as answer]
Hey @rajneesh3923 :wave: 
Did you create an `.env` file or add an environment variable named `DATABASE_URL`?
If not, then you would have to add the above to make it work.

e.g. An `.env` file in the `prisma` folder like this

```
DATABASE_URL=postgresql://username:password@url:5432/prisma2?schema=public
```

**Reply by rajneesh3923** [reply]
Now its working i just added ?schema=public and it starts working. Thanks

**Comment by ludralph** [comment]
Hi @rajneesh3923,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
